toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -550,6 +546,23 @@ os:
     corruptionOption: panic
 ```
 
+When enabling the Verity feature, it is recommended that overlays are applied to
```

Comment:
I suspect that over time the recommendations for verity images could end up being quite long. There are lots of different services/programs available on Azure Linux and a number of them may need special handling under verity. 

In addition, there may be multiple options for a service/program with different tradeoffs. Some users may want a service/program configuration option changeable for convenience. While other users may want it immutable for security.

As such, I recommend splitting off the verity recommendations into its own doc.

Reply:
Sounds reasonable to me.